packages/kbn-securitysolution-autocomplete/src/check_empty_value/index.ts

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Nit: I'd bring more structure to this package by separating components from utils etc. For example:

src/components/autocomplete_field
src/components/autocomplete_field_match
src/utils/fields/check_empty_value
etc

The current flat structure works since there's not too many files, but still a little bit hard to navigate.

Also, file names don't always correspond to component names, for example.

Reply:
I think from this comment and the comment below, maybe we can do a follow up code sweep or discussion around file name changes, conventions?

I have been following mostly patterns seen within these folders to make things portable if we collapse between this package and the others, but I'm open if we want to do a broad cleanup/sweep of just naming changes in 1 faster PR.

Reply:
Yeah, that sounds good, if more people from the team say this kind of restructure/renaming makes sense, doing it in a separate PR either for only this one or across more packages (`kbn-securitysolution-*`?) would be a good idea.